x-pack/plugins/apm/server/lib/transactions/charts/get_anomaly_data/index.ts

```diff
@@ -39,9 +43,16 @@ export async function getAnomalySeries({
     return;
   }
 
-  // don't fetch anomalies if uiFilters are applied
   if (setup.uiFiltersES.length > 0) {
-    return;
+    // filter out known uiFilters like service.environment & service.name
+    const unknownFilters = setup.uiFiltersES.filter(
+      (uiFilter) =>
+        !uiFilter.term?.[SERVICE_ENVIRONMENT] && !uiFilter.terms?.[SERVICE_NAME]
```

Comment:
I wouldn't think service name was part of ui filters?

Reply:
I think it gets included here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/app/TransactionOverview/index.tsx#L101